### ISSUE TITLE
Move downloadable zip delivery to the storage API

### ISSUE THREAD

**Issue by booski** [dsv-su/play#110]
Keeping copies of presentations in zipped format on the app server is redundant and wastes space. The storage API should be extended to provide an endpoint to download a presentation as zip, but this requires a number of changes in the interaction between the app and storage servers:

 - All metadata for a presentation needs to be kept in the package file on the storage server (in addition to in the app server database). The storage server currently only keeps the 'minimal' package data; that being stream, poster and thumbnail data.
 - All metadata changes need to be propagated to the storage server as they happen.

Once the metadata is available and kept current on the storage server, presentations can be exposed as zipped packages at (tentatively) /presentation/zip/<ID>.

Tokens should be used for authenticating zip requests just as with other presentation files.

**Comment by orkit** [comment]
I do not think this is really necessary. The risk is that there is redundant package information on both the play app and the play store. Currently, a download zip is created on demand with the current metadata and embedded in the current player. If it is a storage problem, a solution is not to save the zip file.
